package.json

```diff
@@ -8,21 +8,27 @@
     "url": "https://github.com/elastic/apm-agent-rum-js.git"
   },
   "scripts": {
+    "version": "node scripts/prompt-release",
     "prerelease": "npm test",
     "release": "lerna publish && npm run release-github",
-    "release-package": "node scripts/publish",
+    "release-package": "node scripts/publish-package",
     "release-github": "node scripts/release-github",
     "build-docs": "sh ./scripts/build_docs.sh apm-agent-rum-js ./docs ./build",
     "lint": "npm-run-all --silent -p lint:*",
     "lint:js": "eslint .",
     "lint:license": "node scripts/license-checker",
-    "format": "eslint . --fix",
     "test": "lerna run build:main && lerna run test --stream --scope=$SCOPE --no-prefix",
     "bundlesize": "lerna run bundlesize --stream"
   },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint . --fix",
+      "git add"
```

Comment:
I was thinking about that this might create issues, imagine that you have some modified files (not untracked) that you don't want to include in this commit, e.g. because you want to have separate commits.

Reply:
The way it works is that
1. You add the changes that needs to go in the commit by doing `git add <blah>`
2. Then lint staged gets picked up by precommit hook and then `eslint --fix` and `git add` happens only on those staged files. Modified files that you had wont be affected. So we are safe here.